deblur/workflow.py

```diff
@@ -417,15 +417,15 @@ def remove_artifacts_seqs(seqs_fp,
 
     if coverage_thresh is None:
         if negate:
-            coverage_thresh = 0.95
+            coverage_thresh = 0.95 * 100
         else:
-            coverage_thresh = 0.3
+            coverage_thresh = 0.6 * 100
```

Comment:
Do we know 60% coverage/65% similarity are good for identifying 16S based on gg 88%?